.wordlist.txt

```diff
@@ -189,12 +213,16 @@ NZB
 NZBGet
 occ
 OctoPrint
+octoprint
+odroid
 Odroid
 Odroids
 Ok
+OPcache
 OpenBazaar
 OpenGL
 OpenGLES
+opensource
```

Comment:
Is this rather "open source", i.e. no entry "opensource"?

Reply:
Mostly `open-source` I think: https://en.wikipedia.org/wiki/Open-source_model
So yes, when we remove the entry, the test results tells us where to fix 😄.

Reply:
Ok, deleted "opensource".